website/docs/d/network.html.markdown

```diff
@@ -43,7 +60,7 @@ The following arguments are supported:
   network objects, the ID of the distributed virtual switch for which the port
   group belongs. It is useful to differentiate port groups with same name using
   the distributed virtual switch ID.
-
+* `filter` - (Optional) The type for the discovered network. This is required if you have multiple port groups with the same name. This will be one of `DistributedVirtualPortgroup` for distributed port groups, `Network` for standard (host-based) port groups, or `OpaqueNetwork` for networks managed externally, such as those managed by NSX.
```

Comment:
```suggestion
* `filter` - (Optional) Apply a filter for the discovered network.
   * `network_type`: This is required if you have multiple port groups with the same name. This will be one of `DistributedVirtualPortgroup` for distributed port groups, `Network` for standard (host-based) port groups, or `OpaqueNetwork` for networks managed externally, such as those managed by NSX.
```

Something like this